ols/utils/token_handler.py

```
@@ -125,7 +125,7 @@ def truncate_rag_context(
 
         for node in retrieved_nodes:
 
-            score = float(node.score)
+            score = float(node.get_score())
```

Maybe `get_score(raise_error=False)` to ensure `0` is returned instead of `None`.

---

hmm do we need to put the error under the carpet? but can do it

---

updated @onmete

---

Just thinking it may be safer due to changes in llm libraries in general. :)